server/test/setup.ts

```diff
@@ -18,6 +18,7 @@ if (process.env.DATABASE_URL_TEST) {
 require("@server/database/sequelize");
 
 jest.mock("bull");
+jest.mock("i18next-http-backend");
```

Comment:
This fixed the leak for [`shared/i18n/index.test.ts`](https://github.com/outline/outline/blob/e5c5e8907a52e82356912a5403a6321f2682edd1/shared/i18n/index.test.ts).